src/ldclient_event_process_server.erl

```diff
@@ -67,6 +77,7 @@ start_link(Tag) ->
     {ok, State :: state()} | {ok, State :: state(), timeout() | hibernate} |
     {stop, Reason :: term()} | ignore.
 init([Tag]) ->
+    _Tid = ets:new(ets_table_name(Tag), [set, named_table, {read_concurrency, true}]),
```

Comment:
This theoretically should be a read-heavy operation.

Reply:
If this data were large, then write_concurrency would probably be reasonable, but it is very small.